its_the_way.py

```diff
@@ -0,0 +1,24 @@
+
+import os
+
+
+def file_start(path :str,start:str):
+    """
+    the function get a path of a directory and find the files in the directory that start with start
+    :param path: the path of the directory
+    :param start: the str that the file's names start with
```

Comment:
Also - you should change the parameter name "start" to something better - because it's more a "prefix" than a "start".